sentry-android-core/src/main/java/io/sentry/android/core/DefaultAndroidEventProcessor.java

```diff
@@ -362,6 +370,7 @@ private void setArchitectures(final @NotNull Device device) {
       device.setFreeStorage(getUnusedInternalStorage(internalStorageStat));
     }
 
+    // likely to be applied only if applyScopeData
```

Comment:
the chances that the user opens the app again right away is high, so this won't change much